src/code/z_actor.c

```diff
@@ -457,439 +1269,3557 @@ f32 Actor_XZDistanceToPoint(Actor* actor, Vec3f* point) {
+    if (horse->child != NULL) {
+        return true;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BC7D8.s")
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BC848.s")
+s32 Actor_SetRideActor(GlobalContext* globalCtx, Actor* horse, s32 mountSide) {
+    Player* player = GET_PLAYER(globalCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BC8B8.s")
+    if (!(player->stateFlags1 & 0x003C7880)) {
+        player->rideActor = horse;
+        player->mountSide = mountSide;
+        ActorCutscene_SetIntentToPlay(0x7C);
+        return true;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BCB50.s")
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BCB70.s")
+s32 Actor_NotMounted(GlobalContext* globalCtx, Actor* horse) {
+    if (horse->child == NULL) {
+        return true;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BCBF4.s")
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BCC68.s")
+void func_800B8D10(GameState* gameState, Actor* actor, f32 arg2, s16 arg3, f32 arg4, u32 arg5, u32 arg6) {
+    Player* player = GET_PLAYER(gameState);
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BCCDC.s")
+    player->unk_B74 = arg6;
+    player->unk_B75 = arg5;
+    player->unk_B78 = arg2;
+    player->unk_B76 = arg3;
+    player->unk_B7C = arg4;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BD2B4.s")
+void func_800B8D50(GameState* gameState, Actor* actor, f32 arg2, s16 yaw, f32 arg4, u32 arg5) {
+    func_800B8D10(gameState, actor, arg2, yaw, arg4, 3, arg5);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BD384.s")
+void func_800B8D98(GameState* gameState, Actor* actor, f32 arg2, s16 arg3, f32 arg4) {
+    func_800B8D50(gameState, actor, arg2, arg3, arg4, 0);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BD6B8.s")
+void func_800B8DD4(GameState* gameState, Actor* actor, f32 arg2, s16 arg3, f32 arg4, u32 arg5) {
+    func_800B8D10(gameState, actor, arg2, arg3, arg4, 2, arg5);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BD6E4.s")
+void func_800B8E1C(GameState* gameState, Actor* actor, f32 arg2, s16 arg3, f32 arg4) {
+    func_800B8DD4(gameState, actor, arg2, arg3, arg4, 0);
+}
 
-// This function is very similar to OoT's func_80034A14
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BD888.s")
+void func_800B8E58(Player* player, u16 sfxId) {
+    if (player->currentMask == PLAYER_MASK_GIANT) {
+        func_8019F170(&player->actor.projectedPos, sfxId);
+    } else {
+        Audio_PlaySoundGeneral(sfxId, &player->actor.projectedPos, 4, &D_801DB4B0, &D_801DB4B0, &D_801DB4B8);
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BD9A0.s")
+void Actor_PlaySfxByPos2(Actor* actor, u16 sfxId) {
+    func_8019F1C0(&actor->projectedPos, sfxId);
+}
```

Comment:
```suggestion
void Actor_PlaySfxAtPos(Actor* actor, u16 sfxId) {
    func_8019F1C0(&actor->projectedPos, sfxId);
}
```
Sorry for the flip-flop on this. I'm in the process of doing a more thorough docs of sfx's and using the `AtPos` does seem clearer as others have suggested. There is also no reason for the `2` in this case